webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -77,35 +95,41 @@ class SubscriptionsPage extends Component {
 
   handleDoneTask(taskToPoll) {
     const POLL_TASK_INTERVAL = 5000;
+    const { pollTaskUntilDone, loadSubscriptions } = this.props;
 
-    if (!this.state.polledTask) {
-      this.setState({ polledTask: taskToPoll });
-      this.props.pollTaskUntilDone(taskToPoll.id, {}, POLL_TASK_INTERVAL).then((task) => {
+    pollTaskUntilDone(taskToPoll.id, {}, POLL_TASK_INTERVAL)
+      .then((task) => {
         function getErrors() {
           return (
             <ul>
-              {
-                task.humanized.errors.map(error => <li key={error}> {error} </li>)
-              }
+              {task.humanized.errors.map(error => (
+                <li key={error}> {error} </li>
+              ))}
             </ul>
           );
         }
 
         const message = (
           <span>
-            <span>{__(`Task ${task.humanized.action} completed with a result of ${task.result}`)} </span>
+            <span>
+              {__(`Task ${task.humanized.action} completed with a result of ${
+                  task.result
+                }`)}
```

Comment:
The spacing seems off here.

Also I noticed that the message is poorly formatted (i.e. no punctuation or space between the sentences):

```
Task Refresh Manifest completed with a result of warningClick here to go to the tasks page for the task.
```

This may have been there before your changes but we should try to fix this here if it's easy to do so.  It should ready like this:

```
Task Refresh Manifest completed with a result of warning.  Click here to go to the tasks page for the task.
```

Reply:
The spacing is the result of prettify format, I don't mind to undo